__tests__/app.test.js

```diff
@@ -105,6 +105,25 @@ describe("/api/articles", ()=>{
       expect(body.articles).toBeSortedBy("created_at", { descending: true });
     });
   });
+  test("200: responds with array of articles filtered by the topic mitch", ()=>{
+    return request(app)
+    .get("/api/articles?topic=mitch")
+    .expect(200)
+    .then(({ body })=>{
+      expect(body.articles[0].topic).toBe("mitch");
+      expect(Array.isArray(body.articles)).toBe(true);
+    })
+  })
+  test("400: should return 400 status when given an invalid topic", () => {
+    return request(app)
+    .get("/api/articles?topic=1234")
+    .expect(404);
```

Comment:
This is a `404` test but your description says _400_, because numbers are still counted as TEXT or VARCHAR in sql this is a _valid_ topic, however it hasn't been _found_ which is why it is a `404`